doc/nrf/gs_testing.rst

```diff
@@ -21,7 +21,7 @@ How to connect with the VS Code extension
 
 The `nRF Connect for Visual Studio Code`_ extension is a complete IDE for developing applications for nRF91, nRF53 and nRF52 Series devices.
 It includes nRF Terminal, which is an integrated serial port and RTT terminal to connect to your board.
-For detailed instructions, see `Testing an application`_ in the extension's README file.
+For detailed instructions, see the `nRF Connect for Visual Studio Code`_ documentation site.
```

Comment:
Not sure whether we need two links to the same target in one paragraph.

Reply:
Good call. I changed the first instance to the shortcut tag so it reads the same, but removes the link.

Reply:
I still see two links to the same target.

Reply:
It was updated when I refreshed the preview page.